### DISCUSSION TITLE
Make Sunflower Land a PWA

### DISCUSSION THREAD

**Discussion by rakshit087** [sunflower-land/sunflower-land#1443] [Ideas]
Hey all!
Love playing Sunflower Land. i have a small proposal of making Sunflower Land a progressive web application. 
It will allow users to "Install" sunflower land in their mobiles and desktops which will increase the engagement. 
I don't know if it's possible for web apps using Metamask / Wallets but would be happy to research and raise a PR.

**Comment by adamhannigan** [comment]
Hey @rakshit087 Thanks for raising this 😄 

The blocker right now is that native browser apps (Safari and Chrome) do not support wallets.

There are solutions, but it would require implementing a wallet provider that does not provide custody. (I.e. users login with an email and password).

This is definitely where the future of the game will go to support more people, we are just waiting for the tech to appear. There are hundreds of wallet projects, but they are still improving and quite a lot of friction

**Comment by rakshit087** [comment]
I had similar concerns, but would it work for Mobiles (If not desktops)? Since mobiles have a seperate app for wallet nevertheless.
Also, Web3Auth is a good non custodian wallet provider for future.
